docs/proposals/distributed_architecture/docs/02-Distributed_Scheduling_and_Bidding_Mechanism.md

```diff
@@ -0,0 +1,79 @@
+## Distributed Scheduling and Bidding Mechanism
+
+### Gap Definition and Improvement Objectives
+
+Flintlock currently lacks a distributed scheduling system, relying instead on manual workload allocation per host. Implementing a distributed scheduling mechanism using a bidding process will ensure balanced resource utilization and improved VM provisioning speed.
```

Comment:
This was never going to be part of Flintlock itself, but a part of the wider Liquid Metal.

A reverse bidding process would make the scheduling simpler.